src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -11,6 +11,8 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Where is the orphan introduced in this file? I don't see it.

Reply:
A leftover after fixing the Edmund's review comments, removed, thanks 👍